packages/eslint-config-sheriff/src/getBaseConfig.ts

```diff
@@ -189,26 +190,25 @@ export const getBaseConfig = (
       },
     },
     {
+      extends: [
+        pluginImportX.flatConfigs.recommended,
+        pluginImportX.flatConfigs.typescript,
+      ],
```

Comment:
We only want the "handpicked rules", not their presets. You can remove this.

Reply:
Oh, yep. I misread that code. Makes a lot more sense.